packages/core/src/builder.ts

```diff
@@ -423,15 +423,13 @@ export default class SchemaBuilder<Types extends SchemaTypes> {
       ParentShape<Types, Member>
     >(name);
 
-    options.types.forEach((type) => {
-      verifyRef(type);
```

Comment:
I'd like to leave this here if this is not a function.  The issue this is usually detecting is a ref that's undefined because of a circular import issue (or something similar). Using a thunk should help, but by evaluating here when it's an array, you get a stacktrace that actually points to the right location.

Reply:
Ah nice, will update